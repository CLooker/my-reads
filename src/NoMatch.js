import React from 'react';

const NoMatch = (props) => {
	return (
		<div>
			<h1>404 error</h1>
			<p>The page you requested was not found.</p>
		</div>
	)
}

export default NoMatch;