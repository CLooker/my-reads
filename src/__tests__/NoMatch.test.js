import React from 'react';
import { shallow } from 'enzyme';
import NoMatch from '../components/NoMatch';

describe('NoMatch', () => {
  it('should render properly', () => {
    const component = shallow(<NoMatch />);
    expect(component).toMatchSnapshot();
  });
});
