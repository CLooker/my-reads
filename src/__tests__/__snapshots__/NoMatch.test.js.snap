// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoMatch should render properly 1`] = `
<div>
  <h1>
    404 error
  </h1>
  <p>
    The page you requested was not found.
  </p>
</div>
`;
